### ISSUE TITLE
Remove more self-posting sites with wacky Reach numbers from Awario scores

### ISSUE THREAD

**Issue by cuevasm** [blockstack/app-mining#221]
**What is the problem you are seeing? Please describe.**
A few more sites with the Medium style Reach problem have cropped up. Basically, you can self publish and get way more Reach than really could have possibly been earned. These posts often have no shares or likes and yet can generate thousands in Reach.

**How is this problem misaligned with goals of app mining?**
This score is meant to reflect real press and placements in quality publications.

**What is the explicit recommendation you’re looking to propose?**
Add these sites to the list that don't count:
- https://www.v2ex.com/
- https://qiita.com/
- producthunt (under goals, such as https://www.producthunt.com/@juliet_oberding/goals/59757 - furthermore these seem to count as Web for some and News/Blog for others depending on the content)

**What is the dry run period (if any)**
I'm sure those benefitting will not want to change in period, but an audit period isn't necessarily needed for this. We can easily scrub them immediately.

**Comment by wilsonbright** [comment]
Agreed for removing self-publishing sites. I see top apps in Jan has used Market Watch for Press Release. On searching, I see marketwatch.com press releases are available for 50 dollars in Fiverr. Looks like it is purchasable. Should the News and Blogs to be further split into two categories for reach measurement? or should press releases gets removed?

**Comment by sdsantos** [comment]
Unless we can make this decisions apply to the period they are detected on (which we should, most are mis-categorisations of News/Blogs), Awario score becomes useless and very easily game-able.

Besides the first month of Blended Awareness, the Awario score has been gamed endlessly by 1 or 2 players with dozens of apps. I'm sorry for all the hard work @cuevasm, but if the process stays like this, it's not worth it to have Awario on App Mining. It's too much work for you, and easy bucks for those who make a living out of gaming this.

**Comment by cuevasm** [comment]
I agree with you @sdsantos - I feel like we should make a decision to have an ongoing list where Awario, with my recommendation, can re-categorize or exclude the few links like these that crop up each month within that scoring period. They even admit their algorithm is imperfect (and for normal customers, it doesn't really matter, but we obviously have a little more specific use-case). 

To be frank, I think if you support the best interest/intent of the Awario score and the overall program, you would generally support moving on these rapidly (within the period). If you are more concerned about finding these to help an individual score, then I understand standing behind the protection of having the 1 month warning period (and I don't deny it's a valid strategy). 

In reality, that protection was instituted for bigger, less obvious decisions, not to leave open obvious glitches in the meantime until a rule is passed. That said, I can't simply override that without a community decision on it, so I've scored as normal for this month. I will provide a dry-run sheet in the coming days that omits sites like these.

Last, I don't want to comment on paid stuff now, I have in the past. I generally think paying for placements it's fine and valid marketing - it can only be gamed as far as people can afford and if that's how they want to spend precious startup dollars, so be it. MarketWatch is somewhat respected still and great for SEO.

**Comment by njordhov** [comment]
There are two situations here that should be kept separate:

1. A site is introduced in Awario for the current period, but is determined to be improper for Reach.
2. A site has counted for Reach in previous periods, but is now suddenly scrubbed.

@cuvasm should be free to scrub newly introduced sites that shouldn't be counted, detected as early as possible and with public announcement. No need for a warning or lengthy process, moving fast is good.

However, for sites that already have been considered acceptable in the previous month, there should be a process before they're scrubbed, including advance notice before the period starts.

**Comment by friedger** [comment]
3. sites that counted last period because nobody reviewed them.

These sites should be handled the same way as 1.

Terje Norderhaug <notifications@github.com> schrieb am Mi., 22. Jan. 2020,
22:50:

**Comment by njordhov** [comment]
> 3. sites that counted last period because nobody reviewed them. These sites should be handled the same way as 1.

If so, there should be a public whitelist of definitely accepted sites, a blacklist of the rejected ones, and a graylist of those not reviewed. As is, past month's history of non rejects is the de facto whitelist of sites that miners can assume are acceptable.

**Comment by friedger** [comment]
I'd say it is the de facto grey list

Terje Norderhaug <notifications@github.com> schrieb am Do., 23. Jan. 2020,
22:10:

**Comment by wilsonbright** [comment]
@cuevasm Is there a decision? Were these mentions removed for this month's app mining?

**Comment by cuevasm** [comment]
They won't be removed, we'll have to wait for the next call to move this forward further.
